### PR TITLE
grandpa: remove deprecated afg log target

### PR DESCRIPTION
**Pull request by andresilva**
Missed in https://github.com/paritytech/substrate/pull/12875.

### CODE REVIEW

File: primitives/finality-grandpa/src/lib.rs [hunk 1]
```diff
@@ -322,7 +319,7 @@ impl<H, N> Equivocation<H, N> {
 
 /// Verifies the equivocation proof by making sure that both votes target
 /// different blocks and that its signatures are valid.
-pub fn check_equivocation_proof<H, N>(report: EquivocationProof<H, N>) -> bool
+pub fn check_equivocation_proof<H, N>(report: EquivocationProof<H, N>, log_target: &str) -> bool
```
[the remark this answers is not in the document]
Comment (andresilva) [reply primitives/finality-grandpa/src/lib.rs:322]: I will update the PR to define both log targets in primitives (client and runtime) and then import them from pallet/client code.

[the remark this answers is not in the document]
Comment (bkchr) [reply primitives/finality-grandpa/src/lib.rs:322]: Ahh. I think that isn't that important. I think actually we should maybe do it differently in the runtime and call it `grandpa::runtime`. Then you get all when you just enable grandpa.

[the remark this answers is not in the document]
Comment (andresilva) [reply primitives/finality-grandpa/src/lib.rs:322]: I don't have any strong opinion about it but there seems to be some "standard" of naming runtime log targets as "runtime::*", at least I see a bunch of pallets doing this. I guess the advantage is that you can easily enable all runtime logs of all pallets, whether that is more useful than the reverse (i.e. what you suggested) I don't know 🤷

[the remark this answers is not in the document]
Comment (bkchr) [reply primitives/finality-grandpa/src/lib.rs:322]: Yeah I know that in other pallets we also use the runtime prefix. I don't have a strong opinion either if we should change this. However, I would use a fixed prefix and not pass the prefix as argument

[the remark this answers is not in the document]
Comment (andresilva) [reply primitives/finality-grandpa/src/lib.rs:322]: @davxy I didn't get which side you were arguing for (parameter or no parameter) 🙈 I am fine with dropping the argument, defining both log targets on the primitives crate and then identifying which one to use based on whether we're in std environment or not.

**Review by bkchr — APPROVED** [review]

**Comment by bkchr** [comment]
bot merge

**Pull request merged**